.dockerignore

```
@@ -1,4 +1,7 @@
 .git
+!/.git/HEAD
+!/.git/refs/heads/*
+
```

```suggestion
!/.git/refs/heads/*
```
don't need the empty line, but whatevs :)